content/docs/studio/run-experiments.md

```diff
@@ -0,0 +1,22 @@
+# Run Experiments
+
+You can also run your ML experiments directly from DVC Studio. Your regular
+CI/CD set-up is used to run the experiments. For instance, Github Actions can be
+used to run ML experiments on each new commit.
+
+To run experiments from DVC Studio, select the commit that you want to use and
+click the `Run` button. A form will let you to specify all the changes that you
+want to make to your experiment input files/dirs and parameters.
+
+<img src="/img/studio/cml.png" alt="drawing" width="300"/>
+
+The default values in this form are extracted from your selected commit. For
```

Comment:
For another iteration:

- [ ] Be a bit more detailed in explaining each section? Similar to [/doc/studio/get-started#components-of-a-view](https://dvc-org-studio-docs-vntpxfoqdz.herokuapp.com/doc/studio/get-started#components-of-a-view)

Reply:
Added to new ticket https://github.com/iterative/dvc.org/issues/2468